src/ansible_compat/loaders.py

```diff
@@ -1,10 +1,29 @@
 """Utilities for loading various files."""
+from __future__ import annotations
+
+import os
 from typing import Any
 
 import yaml
 
+from ansible_compat.errors import InvalidPrerequisiteError
+
 
 def yaml_from_file(filepath: str) -> Any:
     """Return a loaded YAML file."""
     with open(filepath, encoding="utf-8") as content:
         return yaml.load(content, Loader=yaml.FullLoader)
+
+
+def fqcn_from_path(filepath: str) -> str | None:
+    """Return a FQCN from a path."""
+    galaxy_file = f"{filepath}/galaxy.yml"
+    if os.path.exists(galaxy_file):
+        galaxy = yaml_from_file(galaxy_file)
+        for k in ("namespace", "name"):
```

Comment:
Do we need to consider the `author` keyword?

It is used in `_get_galaxy_role_ns`
 https://github.com/ansible/ansible-compat/blob/aac5ae2bd68bcf09d1dd7c37066072df30d98c2c/src/ansible_compat/runtime.py#L585-L587

Reply:
No, our code here is only for collections. Author fallback is used on roles.